website/docs/tech-explanation/extreme-decentralization.md

```diff
@@ -0,0 +1,91 @@
+---
+id: extreme-decentralization
+title: Extreme Decentralization
```

Comment:
The title “Extreme Decentralization” may sound negative. Consider alternatives like Uncompromised Decentralization or Maximum Decentralization.